scripts/requirements-ci.txt

```diff
@@ -2,55 +2,56 @@ alabaster==0.7.12
 anytree==2.8.0
 appdirs==1.4.3
 arrow==0.15.5
+astroid==2.4.0
 attrs==19.3.0
 Babel==2.8.0
 breathe==4.14.2
 cached-property==1.5.1
 certifi==2020.4.5.1
 cffi==1.14.0
 chardet==3.0.4
-click==7.1.1
+click==7.1.2
 cmsis-pack-manager==0.2.10
 colorama==0.4.3
 CommonMark==0.5.4
 configobj==5.0.6
-cryptography==2.9
+cryptography==2.9.2
 docopt==0.6.2
 docutils==0.16
 ecdsa==0.15
-editdistance==0.5.3
```

Comment:
you don't need this anymore, or are you sourcing `scripts/requirements-extra.txt` as well ?
Please note, the version in `scripts/requirements-extra.txt` is only `0.5.0`

Reply:
ci doesn't need the packages in extra.txt, they were added by mistake last release

Reply:
thanks for explaining